apps/meteor/app/models/server/models/LivechatInquiry.ts

```diff
@@ -199,11 +200,7 @@ export class LivechatInquiry extends Base {
 					...(department && { department }),
 				},
 			},
-			{
-				$sort: {
-					ts: 1,
-				},
-			},
+			// ...(await this.getSortingQuery({})),
```

Comment:
😬

Reply:
:eyes:

Reply:
deleted the entire function, it was dead code